src/command_modules/azure-cli-ams/HISTORY.rst

```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+2.0.0
++++++
+* GA release.
```

Comment:
There are too many new commands added to call this module GA (it is irrelevant if the service is GA). Please update to 0.3.0 and specifically call out the new command groups and the breaking changes to existing command groups.